### DISCUSSION TITLE
Alt + s,c doesn't work for me at all

### DISCUSSION THREAD

**Discussion by neuromaancer** [helix-editor/helix#7564] [Troubleshooting]
### Summary

I am doing the tutor, however I found `Alt + s,c` doesn't work for me at all. I have checked all the keyboard shortcuts setting, but there is no clash.

### Reproduction Steps

 

I tried this:

1. `hx`

I expected this to happen:

Instead, this happened:

### Helix log

<details><summary>~/.cache/helix/helix.log</summary>

```
please provide a copy of `~/.cache/helix/helix.log` here if possible, you may need to redact some of the lines
```

</details>

### Platform

MACOS

### Terminal Emulator

Iterm2

### Helix Version

helix 23.05 (7f5940be)

**Comment by the-mikedavis** [comment]
Have you configured iterm to treat Option as Alt? https://github.com/helix-editor/helix/issues/2280

**Comment by poliorcetics** [comment]
I use a macOS machine, I can confirm that treating Option as Alt (I recommend setting either right or left as such and leaving the other) works, I've been doing it for quite some time now 😀

**Comment by neuromaancer** [comment]
yes, I have followed the instruction https://github.com/helix-editor/helix/issues/2280 .
But still the same. Below is what I set.

![image](https://github.com/helix-editor/helix/assets/28112871/575838dd-8e2e-45a8-9234-06fd216c31cc)

**Comment by the-mikedavis** [comment]
I can't reproduce this on the latest master or the 23.05 release on iterm2 v3.4.19. Maybe an operating-system-level keybinding is clashing here? Do any key combinations with alt work?

**Reply by neuromaancer** [reply]
I have checked the operating-system-level keybinding, everything is fine.
